errors.go

```diff
@@ -52,6 +52,35 @@ var (
 	// ErrInvalidRunID is returned when the run ID is invalid.
 	ErrInvalidRunID = errors.New("invalid value for run ID")
 
+	// Run Task errors
+
+	// ErrInvalidRunTaskCategory is returned when a run task has a category other than "task"
+	ErrInvalidRunTaskCategory = errors.New(`category must be "tasks"`)
```

Comment:
Is it "tasks" or "task" ? the docs/message disagree

Reply:
tasks, good catch I blame the butterfly keyboard.